Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeViewController.swift

```diff
@@ -664,18 +664,14 @@ class CreateSafeViewController: UIViewController, UITableViewDelegate, UITableVi
     }
 
     private func errorCell(for indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueCell(DetailExpandableTextCell.self, for: indexPath)
+        let cell = tableView.dequeueCell(BasicCell.self, for: indexPath)
         // restrict to 1 tweet length
         let errorText = uiModel.error?.localizedDescription ?? ""
-        let errorPreview = errorText.count <= 144 ? nil : (String(errorText.prefix(144)) + "…")
-        cell.tableView = tableView
-        cell.titleStyle = .error.weight(.medium)
-        cell.expandableTitleStyle = (collapsed: .error, expanded: .error)
-        cell.contentStyle = (collapsed: .error, expanded: .secondary)
-        cell.setTitle("Error")
-        cell.setText(errorText)
-        cell.setCopyText(errorText)
-        cell.setExpandableTitle(errorPreview)
+        let errorPreview = errorText.count <= 144 ? errorText : (String(errorText.prefix(144)) + "…")
```

Comment:
this is not good for the case when the error is bigger than 144 characters, for example when the error contains server response.

We need to find the way to show full content - that's why it was expandable cell before.

Reply:
OK I will then introduce a new type of cell